solr/core/src/test/org/apache/solr/util/TestCircuitBreaker.java

```diff
@@ -41,6 +45,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.hamcrest.CoreMatchers.containsString;
+
+@SuppressWarnings({"rawtypes"})
```

Comment:
Do we have to suppress here? I'd rather not have warnings in the new code to begin with